### PR TITLE
Enable linting for more extensions: `.cjs`, `.mjs`, `.cts`, `.mts`

### PR DESCRIPTION
**Pull request by mrm007**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -123,7 +127,7 @@ const baseConfig = {
   overrides: [
     {
       // TypeScript config
-      files: ['**/*.ts', '**/*.tsx'],
+      files: [`**/*.{${tsExtensions}}`],
```
Comment (michaeltaranto) [inline .eslintrc.js:130]: these arrays need joining yeah?

Comment (mrm007) [reply .eslintrc.js:130]: Nope, JavaScript 🪄 

toString

Comment (askoufis) [reply .eslintrc.js:130]: The template string will call the array's `toString`, which will just result in `"ts,cts,mts,tsx"`.

**Review by askoufis — APPROVED** [review]

File: .eslintrc.js (hunk 1 above)
Comment (michaeltaranto) [reply .eslintrc.js:130]: Sorry, i thought these had to be pipe delimited. My bad

**Review by michaeltaranto — APPROVED** [review]

**Pull request merged**